### ISSUE TITLE
Cyberpunk 2077 Cycling animation when FG enabled (While Driving) 

### ISSUE THREAD

**Issue by ShirouCh** [Nukem9/dlssg-to-fsr3#91]
enabling FG does this after image effect on the car while driving at the bottom of the screen with some wierd lines as well 

enabling Ray reconstruction has this blurry effect in the car model

using RTX 3060ti, game settings set to High/Ultra, RTX medium, DLSS balanced

https://www.youtube.com/watch?v=AAeyfQGkUoM

for reference

**Comment by cichy45** [comment]
Try disabling DLSS upscale or check FSR upscale, users report this is not issue of frame gen but dlss and frame gen amplifies these artifacts.

**Comment by shizoid1** [comment]
I have almost the same thing, only for the place of artifacts there is just a collapse of the frame
https://youtu.be/Ahpm1NDgugA

**Comment by Jackich2** [comment]
disable

> Try disabling DLSS upscale or check FSR upscale, users report this is not issue of frame gen but dlss and frame gen amplifies these artifacts.

Although I checked in Cyberpunk, Still, FSR is buggy. In Alan Wake 2, dlss gives ghosting, but in cyberpunk this is not visible. и ореолы с модом появляются не сразу, а через несколько секунд

**Comment by kecimydo** [comment]
> enabling FG does this after image effect on the car while driving at the bottom of the screen with some wierd lines as well
> 
> enabling Ray reconstruction has this blurry effect in the car model
> 
> using RTX 3060ti, game settings set to High/Ultra, RTX medium, DLSS balanced
> 
> https://www.youtube.com/watch?v=AAeyfQGkUoM
> 
> for reference

DLSS Performance generates noise. it's normal i think.

**Comment by ShirouCh** [comment]
i will it try it out again later, i put the first two in performance just to boost my frames a bit. 

the first 1 was Ray Reconstruction having a noisy blurry things around the vehicle, 

2nd settings i turn it off and it was gone but no more reflections specially noticable in the winshield. 

3rd setting is with the FG mod enabled, which as you can see have the afterimage of the car behind me, lower part of the screen.

**Comment by ShirouCh** [comment]
> I have almost the same thing, only for the place of artifacts there is just a collapse of the frame https://youtu.be/Ahpm1NDgugA

Yes we are seeing similar problems! it is like the lower part of the screen has been generating a separate layer of frames

**Comment by HachemOuanes** [comment]
did anyone find a solution to this yet? I am experiencing the same artifact only when driving!
